sections/main-collection.liquid

```diff
@@ -3,37 +3,25 @@
 
 {%- assign per_page = section.settings.items_per_page -%}
 <div class='yc-product-listing-container'>
-  <div class="breadcrumbs-container">
-    <div class="breadcrumbs-listings">
-      <div class="listings__item">
-        <a href="/" class="first-listing">{{ 'snippets.collection-listing.home' | t }}</a>
-      </div>
-      <div class="listings__item">
-        <img src='{{ 'left-arrow.png' | asset_url }}'>
-      </div>
-      <div class="listings__item">
-        <a href="/collections" class="second-listing">{{ 'snippets.collection-listing.collections' | t }}</a>
-      </div>
-    </div>
-
-    <div class="breadcrumbs-name">
-      <h1>{{ category.name }}</h1>
+  <div class="collection-heading">
+    {%- render 'breadcrumbs' -%}
```

Comment:
We should use the reusable snippet from cod-theme-3

Reply:
in this iteration we want to have the same code as the theme 1 so about the breadcrumbs we already have a ticket for it so we will handle it on the cod theme 1 and theme 2